requirements.txt

```diff
@@ -29,16 +29,16 @@ django-sslserver==0.22                  # Secure HTTP development server
 django-stdimage==5.1.1                  # Advanced ImageField management
 django-test-migrations==1.1.0           # Unit testing for database migrations
 django-user-sessions==1.7.1             # user sessions in DB
-django-weasyprint==1.0.1                # django weasyprint integration
+django-weasyprint==2.1.0                # django weasyprint integration
 djangorestframework==3.12.4             # DRF framework
 django-xforwardedfor-middleware==2.0    # IP forwarding metadata
 flake8==3.8.3                           # PEP checking
 flake8-docstrings==1.6.0                # docstring format testing
 gunicorn>=20.1.0                        # Gunicorn web server
 importlib_metadata                      # Backport for importlib.metadata
-inventree                               # Install the latest version of the InvenTree API python library
```

Comment:
Any idea why that was in here in the first place? Seems to be a non-dependency for deployment enviroments

Reply:
I think it crept in during my testing and never got removed. No real reason to have it here at all.

Reply:
I am currently looking into pip-tools and using full dependency pinning - that might be more stable for testing and deployment. The secondary dependencies could vary the way we do it now.